### PR TITLE
AL-1518 - update to allow tiled image epsg transformer

### PR DESCRIPTION
**Pull request by jtsodapop**
update to allow tiled image epsg transformer to recursively call itself if thre is a list of objects

### CODE REVIEW

File: labelbox/data/annotation_types/data/tiled_image.py [hunk 1]
```diff
@@ -28,6 +28,11 @@
 logger = logging.getLogger(__name__)
 
 
+class AllVectorTools():
+    """Used for the purpose of type hinting"""
+    value: Union[Point, Line, Rectangle, Polygon] = None
```
Comment (msokoloff1) [inline labelbox/data/annotation_types/data/tiled_image.py:33]: You can just do:
`VectorTool = Union[Point, Line,Rectangle, Polygon]`

Comment (jtsodapop) [reply labelbox/data/annotation_types/data/tiled_image.py:33]: Ah got it, thanks. Will update

**Review by Juanbuhler — COMMENTED** [review]
In the [tiled_imagery_basics.ipynb](https://github.com/Labelbox/labelbox-python/pull/437/files#diff-4b8148a02666f252eba0d22c2e9ca0fa863a97186774c9334c9bea4a4c0419e9) notebook, I am getting this in the last cell:

I THINK I am installing the sdk from this branch correctly (pip install . --upgrade in the /data folder)...
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/var/folders/z9/6wpttmtd7qg20hb426w5xkpm0000gn/T/ipykernel_26459/723065197.py in <module>
     11     zoom=15)
     12 
---> 13 annotations = [transformer(Polygon.from_shapely(pixel_polygon)) for pixel_polygon in pixel_polygons]

/var/folders/z9/6wpttmtd7qg20hb426w5xkpm0000gn/T/ipykernel_26459/723065197.py in <listcomp>(.0)
     11     zoom=15)
     12 
---> 13 annotations = [transformer(Polygon.from_shapely(pixel_polygon)) for pixel_polygon in pixel_polygons]

~/Development/venv/lib/python3.7/site-packages/labelbox/data/annotation_types/geometry/polygon.py in from_shapely(cls, shapely_obj)
     38         if not isinstance(shapely_obj, SPolygon):
     39             raise TypeError(
---> 40                 f"Expected Shapely Polygon. Got {shapely_obj.geom_type}")
     41         obj_coords = shapely_obj.__geo_interface__['coordinates'][0]
     42         return Polygon(

TypeError: Expected Shapely Polygon. Got LineString
```

**Comment by Juanbuhler** [comment]
LGTM other than that cell in the fist notebook that fails for me. Let me know if you think that is user error and I'll approve right away.

**Comment by jtsodapop** [comment]
> In the [tiled_imagery_basics.ipynb](https://github.com/Labelbox/labelbox-python/pull/437/files#diff-4b8148a02666f252eba0d22c2e9ca0fa863a97186774c9334c9bea4a4c0419e9) notebook, I am getting this in the last cell:
> 
> I THINK I am installing the sdk from this branch correctly (pip install . --upgrade in the /data folder)...
> 
> ```
> ---------------------------------------------------------------------------
> TypeError                                 Traceback (most recent call last)
> /var/folders/z9/6wpttmtd7qg20hb426w5xkpm0000gn/T/ipykernel_26459/723065197.py in <module>
>      11     zoom=15)
>      12 
> ---> 13 annotations = [transformer(Polygon.from_shapely(pixel_polygon)) for pixel_polygon in pixel_polygons]
> 
> /var/folders/z9/6wpttmtd7qg20hb426w5xkpm0000gn/T/ipykernel_26459/723065197.py in <listcomp>(.0)
>      11     zoom=15)
>      12 
> ---> 13 annotations = [transformer(Polygon.from_shapely(pixel_polygon)) for pixel_polygon in pixel_polygons]
> 
> ~/Development/venv/lib/python3.7/site-packages/labelbox/data/annotation_types/geometry/polygon.py in from_shapely(cls, shapely_obj)
>      38         if not isinstance(shapely_obj, SPolygon):
>      39             raise TypeError(
> ---> 40                 f"Expected Shapely Polygon. Got {shapely_obj.geom_type}")
>      41         obj_coords = shapely_obj.__geo_interface__['coordinates'][0]
>      42         return Polygon(
> 
> TypeError: Expected Shapely Polygon. Got LineString
> ```

I had to install with the following:

!pip install -e '/Users/jonathantso/Documents/gh/lb-py' --upgrade

With that, there are no errors on the last code block

As a sanity check, this is the fix that was applied in this PR:

```if isinstance(shape, Line):
            return Line(points=[self._get_point_obj(p) for p in shape.points])
        if isinstance(shape, Polygon):
            return Polygon(
                points=[self._get_point_obj(p) for p in shape.points])
```

whereas it used to be `if Line or if Polygon...`

**Review by Juanbuhler — APPROVED** [review]
Approving--I still have trouble running one of the notebooks locally but I can run in Colab and jtso can run locally--has to be an issue with my env.

**Pull request merged**